content/applications/inventory_and_mrp/barcode/setup/software.rst

```diff
@@ -1,101 +1,145 @@
+products configuration form. To access a product's form, go to :menuselection:`Inventory app -->
+Products --> Products` and select the product to add a barcode to.
 
-.. image:: software/software_06.png
-    :align: center
+In the :guilabel:`General Information` tab, click the :guilabel:`Barcode` field to either type in
+the barcode or use a scanner to input the barcode value.
+
+.. image:: software/barcode-add-to-product-form.png
+   :alt: The 'Barcode' field on a product form with an active cursor.
 
 .. note::
-         You can easily print the barcode you allocate to the locations via the
-         *Print* menu.
+   If using :doc:`product variants <../../../sales/sales/products_prices/products/variants>`,
+   configure barcodes on individual variants and not the product template to allow scanning to
+   retrieve the variants.
 
-Barcode Formats
-===============
+From Inventory settings
+-----------------------
 
-Most retail products use EAN-13 barcodes, also known as GTIN (Global Trade Identification Numbers).
-GTIN are used by companies to uniquely identify their products and services. While GTIN and UPC are
-often used synonymously, GTIN refers to the number a barcode represents, while UPC refers to the
-barcode itself. More information about GTIN can be found on the GS1 website.
+To access the *Product Barcodes* configuration page, go to :menuselection:`Inventory app -->
+Configuration --> Settings`. In the :guilabel:`Barcode` section, under the :guilabel:`Barcode Scanner` feature, click
+:icon:`fa-arrow-right` :guilabel:`Configure Product Barcodes`. From this list view, click the
+:guilabel:`Barcodes` column for any product to enter its barcode. Barcode scanners will populate
+this field when the product is scanned.
 
-In order to create GTIN for items, a company must have a GS1 Company Prefix. This prefix is the
-number that will appear at the beginning of each GTIN, and will identify the company as the owner of
-the barcode any the products it appears on. To learn more about GS1 Company Prefixes, or purchase a
-license for a prefix, visit the GS1 Company Prefix page.
+.. image:: software/product-barcodes-configure.png
+   :alt: Selecting the Barcode field in the Product Configuration form.
 
-Odoo users are able to use GTIN barcodes to identify their products. However, since Odoo supports
-any numeric string as a barcode, it is also possible to define a custom barcode for internal use.
+.. tip::
+   To filter for the products that do not have barcodes yet, click the :icon:`fa-sort-desc`
+   :guilabel:`(Toggle Search Panel)` icon to add a Custom Filter where the :guilabel:`Barcode`
+   property is :guilabel:`is not set`.
 
-.. _barcode/setup/barcodelookup:
+   .. image:: software/barcode-filter-for-no-barcode.png
+      :alt: 'Add Custom Filter' pop-up with 'Barcode is not set' configured.
 
-Barcode Lookup
-==============
+.. _barcode/setup/location:
+
+From Barcode app
```

Comment:
Consider *leading* with this instruction and gracefully falling back to the alternative options after, by order of manual entry. It gives readers the option to skip redundant methods after leading with the most relevant one.